org.easyrpg.player.yml

```diff
@@ -0,0 +1,97 @@
+app-id: org.easyrpg.player
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: easyrpg-player
+rename-desktop-file: easyrpg-player.desktop
+rename-appdata-file: easyrpg-player.metainfo.xml
+rename-icon: easyrpg-player
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --filesystem=home # user can store games anywhere
+  - --device=all # needed for gamepad access
+modules:
+  - name: liblcf
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DLIBLCF_ENABLE_TOOLS=OFF
+      - -DLIBLCF_WITH_XML=OFF
+      - -DLIBLCF_ENABLE_TESTS=OFF
+      - -DLIBLCF_UPDATE_MIMEDB=OFF
+    sources:
+      - type: git
+        url: https://github.com/EasyRPG/liblcf.git
+        tag: '0.8'
+        commit: 9c09529e3de0a4106470082ef48a00d4c3593c33
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+      - /lib/cmake
+      - /share
+  - name: libfmt
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DFMT_TEST=OFF
+      - -DBUILD_SHARED_LIBS=TRUE
+    sources:
+      - type: git
+        url: https://github.com/fmtlib/fmt.git
+        tag: '9.1.0'
+        commit: a33701196adfad74917046096bf5a2aa0ab0bb50
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+      - /lib/cmake
+  # this is copied from shared-modules to not add the dependency on it
```

Comment:
why not?

Reply:
Just to have no additional submodule step, I would say personal preference.
I could amend this if wanted.